app/src/test/java/org/simple/clinic/summary/addphone/AddPhoneNumberDialogControllerTest.kt

```diff
@@ -79,13 +80,10 @@ class AddPhoneNumberDialogControllerTest {
   }
 
   @Test
-  @Parameters(method = "validation errors")
-  fun `when save is clicked, the number should not be saved if it's invalid`(
-      validationError: Result
-  ) {
+  fun `when save is clicked, the number should not be saved if it's invalid`() {
     // given
     val newNumber = "123"
-    whenever(validator.validate(newNumber, type = LANDLINE_OR_MOBILE)).thenReturn(validationError)
+    whenever(validator.validate(newNumber, type = LANDLINE_OR_MOBILE)).thenReturn(validationErrors.random())
```

Comment:
Can we use a real instance of `PhoneNumberValidator` here? See step 3 in [`Stabilize`](https://github.com/simpledotorg/simple-android/blob/master/doc/mobius/migrating-to-mobius.md#steps).